gp-auto-extract.php

```diff
@@ -77,241 +153,501 @@ public function auto_extract( $project_path ) {
 		}
 
 		gp_notice_set( $message );
-		
+
 		// Redirect back to the project home.
 		wp_redirect( $url );
 	}
-	
-	// This function adds the "Auto Extract" option to the projects menu.
+
+	/**
+	 * This function is here as placeholder to support adding the auto extract option to the router.
+	 * Without this placeholder there is a fatal error generated.
+	 */
+	public function after_request() {
```

Comment:
This placeholder function was removed in the current tree, you must have added it back in by accident during the rebase.